space-cli/cmd/modules/services/delete_test.go

```diff
@@ -528,3 +528,1065 @@ func Test_deleteSecret(t *testing.T) {
+					args: []interface{}{
+						http.MethodGet,
+						"/v1/runner/myproject/services",
+						map[string]string{},
+						new(model.Response),
+					},
+					paramsReturned: []interface{}{
+						nil,
+						model.Response{
+							Result: []interface{}{
+								map[string]interface{}{
+									"id":      "localAdmin",
+									"version": "v1",
+								},
+								map[string]interface{}{
+									"id":      "local",
+									"version": "v1",
+								},
+							},
+						},
+					},
+				},
+				{
+					method: "MakeHTTPRequest",
+					args: []interface{}{
+						http.MethodDelete,
+						"/v1/runner/myproject/services/localAdmin/v1",
+						map[string]string{},
+						new(model.Response),
+					},
+					paramsReturned: []interface{}{
+						errors.New("bad request"),
+						model.Response{
+							Result: []interface{}{
+								map[string]interface{}{
+									"statusCode": 400,
+								},
+							},
+						},
+					},
+				},
+			},
+			surveyMockArgs: []mockArgs{
+				{
+					method: "AskOne",
+					args: []interface{}{
+						&survey.Select{
+							Message: "Choose the resource ID: ",
+							Options: []string{"localAdmin::v1", "local::v1"},
+							Default: []string{"localAdmin::v1", "local::v1"}[0],
+						},
+						&surveyNoMatchReturnValue,
+					},
+					paramsReturned: []interface{}{nil, "localAdmin::v1"},
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "Prefix does not match any services and service successfully deleted",
+			args: args{project: "myproject", prefix: map[string]string{"serviceId": "b"}},
+			transportMockArgs: []mockArgs{
+				{
+					method: "MakeHTTPRequest",
+					args: []interface{}{
+						http.MethodGet,
+						"/v1/runner/myproject/services",
+						map[string]string{},
+						new(model.Response),
+					},
+					paramsReturned: []interface{}{
+						nil,
+						model.Response{
+							Result: []interface{}{
+								map[string]interface{}{
+									"id":      "localAdmin",
+									"version": "v1",
+								},
+								map[string]interface{}{
+									"id":      "local",
+									"version": "v1",
+								},
+							},
+						},
+					},
+				},
+				{
+					method: "MakeHTTPRequest",
+					args: []interface{}{
+						http.MethodDelete,
+						"/v1/runner/myproject/services/localAdmin/v1",
+						map[string]string{},
+						new(model.Response),
+					},
+					paramsReturned: []interface{}{
+						nil,
+						model.Response{
+							Result: []interface{}{
+								map[string]interface{}{
+									"statusCode": 200,
+								},
+							},
+						},
+					},
+				},
+			},
+			surveyMockArgs: []mockArgs{
+				{
+					method: "AskOne",
+					args: []interface{}{
+						&survey.Select{
+							Message: "Choose the resource ID: ",
+							Options: []string{"localAdmin::v1", "local::v1"},
+							Default: []string{"localAdmin::v1", "local::v1"}[0],
+						},
+						&surveyNoMatchReturnValue,
+					},
+					paramsReturned: []interface{}{nil, "localAdmin::v1"},
+				},
+			},
+		},
+	}
```

Comment:
Add some test cases for 

- Exact match
- User has provided both service id & version
- Add more services in, Get services HTTP call